ttnn/cpp/ttnn/operations/pool/upsample/upsample.cpp

```diff
@@ -27,21 +27,8 @@ ttnn::Tensor ExecuteUpSample::invoke(
                 scale_h = sf;
                 scale_w = sf;
             } else if constexpr (std::is_same_v<T, tt::tt_metal::Array2D>) {
-                scale_w = sf.at(0);
-                int scale_c = sf.at(1);
-                TT_FATAL(scale_c == 1, "Error");
-            } else if constexpr (std::is_same_v<T, tt::tt_metal::Array3D>) {
                 scale_h = sf.at(0);
-                scale_w = sf.at(1);
-                int scale_c = sf.at(2);
-                TT_FATAL(scale_c == 1, "Error");
-            } else if constexpr (std::is_same_v<T, tt::tt_metal::Array4D>) {
-                int scale_n = sf.at(0);
-                scale_h = sf.at(1);
-                scale_w = sf.at(2);
-                int scale_c = sf.at(3);
-                TT_FATAL(scale_n == 1, "Error");
-                TT_FATAL(scale_c == 1, "Error");
+                scale_w = sf.at(0);
```

Comment:
Please fix this before merging, this should be `sf.at(1)`.

Reply:
Done, Thanks.